onyo/lib/__init__.py

```diff
@@ -1,7 +1,9 @@
+from .filters import Filter, OnyoInvalidFilterError
 from .onyo import Repo, OnyoInvalidRepoError, OnyoProtectedPathError
 
 __all__ = [
+    'Filter',
     'Repo',
+    'OnyoInvalidFilterError',
     'OnyoInvalidRepoError',
-    'OnyoProtectedPathError',
-]
```

Comment:
It is PEP8 compliant though, but regarding the trailing comma I will change it back because that's a good point. Note that I'm not going around changing everyone's code style, I added the `InvalidFilterError` there (at first at the bottom, then alphabetized) so it's a remnant from that.